File: src/main/java/com/thebluehats/server/game/commands/GiveObsidianCommand.java
package com.thebluehats.server.game.commands;

import org.bukkit.Material;
import org.bukkit.entity.Player;
import org.bukkit.inventory.ItemStack;

public class GiveObsidianCommand extends GameCommand {
    @Override
    public String[] getCommandNames() {
        return new String[] { "giveobsidian" };
    }

    @Override
    public String getUsageMessage(String cmd) {
        return formatStandardUsageMessage(cmd, "Gives a stack of obsidian.");
    }

    @Override
    public void runCommand(Player player, String commandName, String[] args) {
        ItemStack obsidian = new ItemStack(Material.OBSIDIAN, 64);

        player.getInventory().addItem(obsidian);
    }
}

File: src/main/java/com/thebluehats/server/game/commands/GiveObsidianCommand.kt
package com.thebluehats.server.game.commands

import org.bukkit.Material
import org.bukkit.entity.Player
import org.bukkit.inventory.ItemStack

class GiveObsidianCommand : GameCommand() {
    override val commandNames: Array<String>
        get() = arrayOf("giveobsidian")

    override fun getUsageMessage(cmd: String?): String? {
        return formatStandardUsageMessage(cmd!!, "Gives a stack of obsidian.")
    }

    override fun runCommand(player: Player, cmd: String?, args: Array<String>) {
        val obsidian = ItemStack(Material.OBSIDIAN, 64)
        player.inventory.addItem(obsidian)
    }
}